github/repositories/main.tf

```diff
@@ -1,12 +1,44 @@
-resource "github_repository" "repo" {
-  for_each    = var.repos
-  name        = each.value["name"]
-  description = each.value["description"]
-  visibility  = each.value["visibility"]
+module "github_repository" {
+  source  = "mineiros-io/repository/github"
+  version = "~> 0.16.0"
 
-  template {
+  for_each               = { for key, value in var.repositories : key => value }
+  name                   = each.value.name
+  description            = each.value.description
+  visibility             = each.value.visibility
+  has_issues             = true
+  has_projects           = true
+  has_wiki               = false
+  allow_merge_commit     = false
+  allow_rebase_merge     = false
+  allow_squash_merge     = true
+  allow_auto_merge       = false
+  has_downloads          = false
+  auto_init              = true
+  delete_branch_on_merge = true
+  license_template       = "mit"
+  topics                 = [each.value.squad]
+  vulnerability_alerts   = true
+
+  template = {
     owner      = "nmbrs"
```

Comment:
Please, consider assigning owner value to `var.github_owner` instead of the hardcoded organization value
```suggestion
  template = {
    owner      = var.github_owner
```

Reply:
this is related with the template